apps/new-dealer/translations/src/en/validation.json

```diff
@@ -1,4 +1,16 @@
 {
+  "company": {
+    "required": "Please select whether you're applying on behalf of a company or a sole trader"
```

Comment:
@joefitter I didn't know that! These validation messages are by me and not done by a designer